src/main/java/edu/ucsb/ucsbcslas/controllers/TutorNotesController.java

```diff
@@ -0,0 +1,275 @@
+            newNotes.setTutor(tutor.get());
+        } else {
+            return getIncorrectInputResponse();
+        }
+        newNotes.setMessage(ta.getString("NotesType"));
+
+        logger.info("newNotes= {}", newNotes);
+        TutorNotes savedTutorNotes = tutorNotesRepository.save(newNotes);
+        String body = mapper.writeValueAsString(savedTutorNotes);
+        return ResponseEntity.ok().body(body);
+    }
+
+    @GetMapping(value = "/api/member/tutorNotes", produces = "application/json")
+    public ResponseEntity<String> getTutorNotes(@RequestHeader("Authorization") String authorization)
+            throws JsonProcessingException {
+        List<TutorNotes> tutorNotesList = new ArrayList();
+        if (authControllerAdvice.getIsAdmin(authorization)) {
+            tutorNotesList = tutorNotesRepository.findAll();
+            if (tutorNotesList.isEmpty()) {
+                return ResponseEntity.notFound().build();
+            }
+            ObjectMapper mapper = new ObjectMapper();
+            String body = mapper.writeValueAsString(tutorNotesList);
+            return ResponseEntity.ok().body(body);
+        } else {
+            List<Course> courseList = courseRepository
+                    .findAllByInstructorEmail(authControllerAdvice.getUser(authorization).getEmail());
+            if (!(courseList.isEmpty())) {
+                for (Course temp : courseList) {
+                    Optional<Course> course = courseRepository.findById(temp.getId());
+                    if (course.isPresent()) {
+                        List<TutorNotes> tutorNotes = tutorNotesRepository
+                                .findAllByCourse(course.get());
+                        tutorNotesList.addAll(tutorNotes);
+                    }
+                }
+                if (tutorNotesList.isEmpty()) {
+                    return ResponseEntity.notFound().build();
+                }
+                ObjectMapper mapper = new ObjectMapper();
+                String body = mapper.writeValueAsString(tutorNotesList);
+                return ResponseEntity.ok().body(body);
+            }
+        }
+        if (authControllerAdvice.getIsMember(authorization)) {
+            Optional<Tutor> tutor = tutorRepository.findByEmail(authControllerAdvice.getUser(authorization).getEmail());
+            if (tutor.isPresent()) {
+                List<TutorNotes> tutorNotes = tutorNotesRepository.findAllByTutor(tutor.get());
+                tutorNotesList.addAll(tutorNotes);
+            }
+            if (tutorNotesList.isEmpty()) {
+                return ResponseEntity.notFound().build();
+            }
+            ObjectMapper mapper = new ObjectMapper();
+            String body = mapper.writeValueAsString(tutorNotesList);
+            return ResponseEntity.ok().body(body);
+        }
+        return getUnauthorizedResponse("member");
+    }
+    
+    @GetMapping(value = "/api/public/tutorNotes/{course_id}", produces = "application/json")
```

Comment:
This should at least be an `/api/member` endpoint available only to members.     It should definitely NOT be a public endpoint.

Eventually, we probably need to implement even more security: tutor notes should be available to be read only by tutors of the specific course, instructors and admins.   That's too much for this particular issue.  But at the very least, we should make this a non-public endpoint.